modules/RTC/RTC.js

```diff
@@ -157,6 +158,12 @@ export default class RTC extends Listenable {
             };
             this.addListener(RTCEvents.DATA_CHANNEL_OPEN,
                 this._dataChannelOpenListener);
+
+            this.addListener(RTCEvents.LASTN_ENDPOINT_CHANGED,
+                lastNEndpoints => {
+                    this._lastNEndpoints = lastNEndpoints;
```

Comment:
where is `this._lastNEndpoints` initialized? If we initialize it to [] in the constructor that would simplify the check next...